Resources/public/js/views/ez-fieldeditview.js

```diff
@@ -394,6 +394,17 @@ YUI.add('ez-fieldeditview', function (Y) {
             },
 
             /**
+             * The logged in user
+             *
+             * @attribute user
+             * @type {eZ.User}
+             *  @default null
```

Comment:
why do you set the default value to `null` here while you document it as `undefined` in the Author edit view and in others cases there's no default value and the attribute is marked as required ?
IMHO, you should make the attribute as required all the way down and don't document nor set any default value.